app/ante/panic_test.go

```diff
@@ -0,0 +1,37 @@
+package ante_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/ante"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPanicHandlerDecorator(t *testing.T) {
```

Comment:
[optional] do we want to include a test case that verifies behavior of the `NewSetUpContextDecorator` is preserved? In other words,  a tx that runs out of gas still results in an `ErrOutOfGas`. I expect it will pass b/c that ante handler would catch an OutOfGasPanic before the panic handler decorator.

Reply:
Yeah the fact that the `PanicHandlerDecorator` comes before means all behaviour afterwards should theoretically be the same